appveyor.yml

```diff
@@ -0,0 +1,69 @@
+# https://ci.appveyor.com/project/spyder-ide/spyder-terminal
+
+environment:
+
+  global:
+    PYTHON: "C:\\conda"
+    MINICONDA_VERSION: "latest"
+    CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\ci-helpers\\appveyor\\windows_sdk.cmd"
+    PIP_DEPENDENCIES_FLAGS: "-q"
+    CONDA_DEPENDENCIES_FLAGS: "--quiet"
+    CONDA_DEPENDENCIES: >
+      spyder pytest pytest-cov
+    PIP_DEPENDENCIES: "pytest-qt pytest-timeout flaky coloredlogs pytest-xvfb"
+    APPVEYOR_RDP_PASSWORD: "dcca4c4863E30d56c2e0dda6327370b3#"
+
+  matrix:
+    - PYTHON_VERSION: "3.6"
+      PYTHON_ARCH: "32"
+      ARCH: "x86"
+      platform: "x86"
+    - PYTHON_VERSION: "3.5"
+      PYTHON_ARCH: "32"
+      ARCH: "x86"
+      platform: "x86"
+    - PYTHON_VERSION: "3.6"
+      PYTHON_ARCH: "64"
+      ARCH: "amd64"
+      platform: "x64"
+    - PYTHON_VERSION: "3.5"
+      PYTHON_ARCH: "64"
+      ARCH: "amd64"
+      platform: "x64"
+
+platform:
+  -x64
+
+install:
+  # If there is a newer build queued for the same PR, cancel this one.
+  # The AppVeyor 'rollout builds' option is supposed to serve the same
+  # purpose but it is problematic because it tends to cancel builds pushed
+  # directly to master instead of just PR builds (or the converse).
+  # credits: JuliaLang developers.
+  - ps: if ($env:APPVEYOR_PULL_REQUEST_NUMBER -and $env:APPVEYOR_BUILD_NUMBER -ne ((Invoke-RestMethod `
+      https://ci.appveyor.com/api/projects/$env:APPVEYOR_ACCOUNT_NAME/$env:APPVEYOR_PROJECT_SLUG/history?recordsNumber=50).builds | `
+      Where-Object pullRequestId -eq $env:APPVEYOR_PULL_REQUEST_NUMBER)[0].buildNumber) { `
+        throw "There are newer queued builds for this pull request, failing early." }
+  - "git clone git://github.com/astropy/ci-helpers.git"
+  - "powershell ci-helpers/appveyor/install-miniconda.ps1"
+  - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
+  - "activate test"
+  - "conda install -q -y pywinpty coveralls flaky -c conda-forge"
```

Comment:
* There's no need for `flaky` to be here because it's already installed by pip
* No need for coveralls either because Circle is uploading coverage already.
* Please move `pywinpty` to the `spyder-ide` channel and install it from there. We must not install things from conda-forge to avoid conflicting dependencies with the default channels.